website/src/components/ThreeUpText.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FunctionComponent } from 'react'
+
+interface Item {
+    icon?: React.ReactNode
+    subtitle: React.ReactNode
+    description: string
+}
+
+interface Props {
+    title: string
+    items: Item[]
+}
+
+export const ThreeUpText: FunctionComponent<Props> = ({ title, items }) => (
+    <section className="row mx-lg-0 mx-4">
+        <h1 className="text-center col-12 max-w-lg-550 mx-auto mt-6 mb-lg-4 px-0 font-weight-bold">{title}</h1>
+
+        <div className="mb-6 d-flex flex-wrap justify-content-between">
+            {items.map((item, index) => (
+                <div
+                    key={`item-${index + 1}-${item.description}`}
+                    className="col-12 col-xl-3 min-w-350 text-center pt-5"
```

Comment:
Nice catch! I think the `min-w-350` needs to stay b/c otherwise the sections get very squished on a larger screen. @bretthayes 
<img width="1229" alt="Screen Shot 2022-04-26 at 9 40 40 AM" src="https://user-images.githubusercontent.com/59381432/165344280-9d5abd19-b761-4bb8-86eb-5bb72bfc9130.png">
VS
<img width="1244" alt="Screen Shot 2022-04-26 at 10 06 44 AM" src="https://user-images.githubusercontent.com/59381432/165344319-3e89d321-d79c-40e3-8106-631783558a61.png">

Reply:
I see what you mean. I anticipate QA may have comments about the subtitle words breaking to different lines. But I agree, keeping all 3 horizontal on `lg` feels more important.